pmm-app/src/pmm-check/components/AllChecksTab/AllChecksTab.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useEffect, useState } from 'react';
+import { cx } from 'emotion';
+import { CheckDetails } from 'pmm-check/types';
+import { CheckService } from 'pmm-check/Check.service';
+import { Spinner, useTheme } from '@grafana/ui';
+// TODO: make a table shared style
```

Comment:
Is there a ticket, or you plan on doing it now?

Reply:
No at both, we need a ticket :)